metron-deployment/roles/metron_streaming/tasks/metron_topology.yml

```diff
@@ -16,7 +16,7 @@
 #
 ---
 - name: Submit Metron Parser topologies
```

Comment:
Based on recent changes, we no longer submit the topologies as part of the deployment process. The topologies (and most other services) are started at the end of the deployment only if the user has configured them to start.

Based on the new arguments, we'll need updates in the monit space.  I'm thinking at least these will be impacted.
- `metron-deployment/roles/monit/templates/monit/parsers.monit`
- `metron-deployment/roles/monit/templates/scripts/start_topology.sh`

Let me know if you'd like help tackling that.